docs/maintainers/maintainers-guide.md

```diff
@@ -70,39 +70,39 @@ There are two ways to add contributors to a Contributor Insight Page:
 1. Create a new Contributor Insight Page with selected contributors.
 2. Add selected contributors to an existing Contributor Insight Page.
 
-## Contributor Insights : Connecting with Contributors
+## Contributor Insights: Connecting with Contributors
 
 OpenSauced Contributor Insights feature allows you to categorize, monitor, and analyze various groups of contributors. You can use Contributor Insights to track the contributions of a specific team, a specific repository, or a group of repositories. This feature offers granular insights into each contributor's activities and contributions, helping maintainers to track individual contribution histories, identify active engagement, recognize new or alumni contributors, and compare the performance of different contributors.
 
 The Contributor Insights features allow you to:
 
 - track the commits and PRs of the contributors on your list,
 - filter your contributors by their activity level, including most active, new, and alumni,
-- view the repositories that your contributors have contributed to, the top programming languages they've used, and their time zones.
+- view the repositories your contributors have contributed to, the top programming languages they've used, and their time zones.
```

Comment:
[improvement] the wording here could be clearer. Thoughts?
```suggestion
- view the repositories your contributors have worked on, the top programming languages they've used, and their time zones.
```

Reply:
I think "contributed to" is make much more sense here.

Reply:
I don't know @adiati98. Saying "contributors have contributed to" does sound a bit redundant, so using a different word like "worked on" conveys the same point but in a clearer way. What do you think @BekahHW?